src/Dfc.CourseDirectory.Web/appsettings.Environment.PreProduction.json

```diff
@@ -3,7 +3,7 @@
     "ApiBaseUri": "https://pp-api.signin.education.gov.uk/",
     "ServiceId": "724D02EB-221E-4638-9E01-9B49B1E67750"
   },
-  "FeatureFlags": "",
+  "FeatureFlags": "TLevelsBeta",
```

Comment:
As discussed with Steve, enabled in PreProd only for testing in isolation and in an environment where we can confirm that T Levels aren't published by FAC